### ISSUE TITLE
Route navigation does not work inside the viewport

### ISSUE THREAD

**Issue by mlvtito** [aurelia/aurelia#1282]
# 🐛 Bug Report

I have an app with a main viewport. Inside this main viewport, I have another viewport named "sub".

As long as I use links to navigate in the same component as the viewport, everything is working fine. But If I try to navigate to another component with a link inside the viewport, an exception is raised.

Here is a sample application to reproduce this behavior: https://github.com/mlvtito/aurelia-router-bug

## 🤔 Expected Behavior

Navigation links should work from any component, in any viewport.

## 😯 Current Behavior

In the sample application, if I click on the "Go to another sub content" link, this exception is raised:
```
Uncaught (in promise) Error: Failed to resolve VR(viewport:'sub',component:'another-sub-content',resolution:'dynamic',append:false) at:
RC(path:'my-app',viewports:[VPA(state:currIsActive|nextIsScheduled,plan:'none',resolution:'dynamic',n:RN(ctx:'my-app/first-menu',c:'first-menu',children:[RN(ctx:'my-app/first-menu/first-sub-menu',c:'first-sub-menu')]),c:RN(ctx:'my-app/first-menu',c:'first-menu',children:[RN(ctx:'my-app/first-menu/first-sub-menu',c:'first-sub-menu')]),viewport:VP(ctx:'my-app',name:'default',default:'first-menu'))])
 RC(path:'my-app/first-menu',viewports:[VPA(state:currIsActive|nextIsScheduled,plan:'none',resolution:'dynamic',n:RN(ctx:'my-app/first-menu/first-sub-menu',c:'first-sub-menu'),c:RN(ctx:'my-app/first-menu/first-sub-menu',c:'first-sub-menu'),viewport:VP(ctx:'my-app/first-menu',name:'sub'))])
  RC(path:'my-app/first-menu/first-sub-menu',viewports:[])
    resolveViewportAgent index.js:3384
    It index.js:2178
    Ct index.js:2047
    Rt index.js:1997
    Et index.js:1939
    Et index.js:1939
    E index.js:292
    Et index.js:1936
    Et index.js:1947
    Et index.js:1947
    E index.js:292
    Et index.js:1936
    Et index.js:1947
    Et index.js:1947
    E index.js:292
    Et index.js:1936
    yt index.js:1927
    E index.js:292
    yt index.js:1927
    run index.js:2590
    run index.js:2353
    run index.js:2588
    enqueue index.js:2537
    load index.js:2470
    onClick index.js:3594
    handleEvent index.js:4023
    Yt index.js:3995
    DelegateSubscription index.js:4034
    addEventListener index.js:4075
    binding index.js:3601
    activate index.js:3072
    Dt index.js:3179
    bind index.js:3116
    activate index.js:3083
    activate index.js:3691
    E index.js:292
    activate index.js:3691
    E index.js:292
    activate index.js:3691
    Ei index.js:8887
    E index.js:292
    start index.js:8883
    <anonymous> main.ts:13
    ts entry.bundle.js:402
    __webpack_require__ entry.bundle.js:675
    <anonymous> entry.bundle.js:751
    <anonymous> entry.bundle.js:754
index.js:3384
```

## 💁 Possible Solution

No idea. I'm not inot Aurelia code yet ;)

## 🔦 Context

Basically, I'm trying to add buttons to edit or create items next to items list. I don't know how to do that with this bug.

## 💻 Code Sample

https://github.com/mlvtito/aurelia-router-bug

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Aurelia          | 2.0.0-alpha.16
| Language         | typescript
| Browser          | Firefox 90
| Bundler          | Webpack 5.39.1
| Operating System | Ubuntu 21.04
| NPM/Node/Yarn    | npm
